packages/base/src/signals/signal-manager.ts

```diff
@@ -41,7 +41,8 @@ export const Signals = {
     RESET_ZOOM: 'reset zoom',
     MARKER_CATEGORIES_FETCHED: 'marker categories fetched',
     MARKERSETS_FETCHED: 'markersets fetched',
-    MARKER_CATEGORY_CLOSED: 'marker category closed'
+    MARKER_CATEGORY_CLOSED: 'marker category closed',
+    CHECK_OPEN_WIDGETS: 'check open widgets'
```

Comment:
There's too many signals already doing the same thing, I'm a bit confused.

Could you see if you can reuse one of these existing signals:
    EXPERIMENT_CLOSED: 'experiment closed',
    OPENED_TRACES_UPDATED: 'opened traces updated',
    CLOSE_TRACEVIEWERTAB: 'tab closed',

Whichever one is fired in all the cases you care about...

Reply:
> There's too many signals already doing the same thing, I'm a bit confused.
> 
> Could you see if you can reuse one of these existing signals: EXPERIMENT_CLOSED: 'experiment closed', OPENED_TRACES_UPDATED: 'opened traces updated', CLOSE_TRACEVIEWERTAB: 'tab closed',
> 
> Whichever one is fired in all the cases you care about...

Reworked the code to implement OPENED_TRACES_UPDATED signal